src/CMakeLists.txt

```diff
@@ -52,6 +52,7 @@ target_link_libraries(avitab_common
 if(WIN32)
     target_link_libraries(avitab_common
         ws2_32
+        bcrypt
```

Comment:
Does this library ship with Windows or is it a new runtime dependency? In the latter case, that won't work unfortunately: Every dependency must be linked statically.

Reply:
The updated curl library introducers the dependency on bcrypt. The inclusion of this library seems to be implicit on Linux and MacOS. On Windows bcrypt needs to be listed explicitly as a dependency.

I took a look at XP12 using ProcessExplorer, and it shows that the bcrypt library being loaded from the Windows distribution (C:\Windows\System32\bcrypt.dll) The library is loaded even when Avitab is not installed as a plugin. When Avitab is added as a plugin then it works as expected, even though bcrypt is a DLL and not a statically linked library.

I think the dependency is OK.

Reply:
Thanks for the check!